t/t3431-rebase-ignore-ws.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+#
+# Copyright (c) 2019 Rohit Ashiwal
+#
+
+test_description='git rebase --ignore-whitespace tests'
+
+. ./test-lib.sh
+
+test_expect_success 'setup' '
+	git checkout -b topic &&
+	q_to_tab >main.c <<-EOF &&
+	int main() {
+	Qprintf("Hello World\n");
+	Qreturn 0;
+	}
+	EOF
+	git add main.c &&
+	git commit -m "add main.c" &&
+	q_to_tab >main.c <<-EOF &&
+	int main() {
+	Qprintf("Hello Git\n");
+	Qreturn 0;
+	}
+	EOF
+	git commit -am "update main.c" &&
+	git tag side &&
+
+	git checkout --orphan master &&
+	cat >main.c <<-EOF &&
+	int main() {
+	        printf("Hello World\n");
+	        return 0;
+	}
+	EOF
+	git add main.c &&
+	git commit -m "add main.c" &&
+	git tag main
+'
+
+test_expect_success '--ignore-whitespace works with am backend' '
+	test_must_fail git rebase main side &&
```

Comment:
Is there any output we can check to make sure this failed the way we expected?

Reply:
:thinking: yes, we can.

Reply:
I checked it and `git rebase main side` does not uniquely produce anything which relates it to `whitespace` errors.